package.json

```diff
@@ -97,7 +97,7 @@
     "lint:js": "eslint ./lib ./test/*.js ./docker/**/*.js ./test/framework/*.js ./test/lib/*.js ./test/integration/** ./test/unit/** ",
     "lint:ts": "eslint ./tsdef/*.ts ./lib/**/*.ts",
     "lint": "npm-run-all lint:js lint:ts",
-    "test:unit": "npm-run-all test:unit:js test:unit:ts",
+    "test:unit": "npm-run-all build:es5 test:unit:js test:unit:ts",
```

Comment:
Why do we need to build es5 for unit tests?

Reply:
That is because our integration and unit tests share some shared libraries. Any library used by integration test cannot use "source" files - It must `require es5` version. And same libraries are used by unit tests, we must build es5 before unit tests.